app/helpers/plate_helper.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 module PlateHelper # rubocop:todo Style/Documentation
-  class WellFailingPresenter < BasicObject # rubocop:todo Style/Documentation
+  # Proxy object wrapping the form alongside the presenter.
+  # This allows us to use the shared plate partial, but pass the form
+  # object through to the custom aliquot partial
+  class WellFailingPresenter < BasicObject
     def initialize(form, presenter)
-      @form = form # rubocop:todo Rails/HelperInstanceVariable
-      @_presenter = presenter # rubocop:todo Rails/HelperInstanceVariable
+      @form = form # rubocop:disable Rails/HelperInstanceVariable
+      @_presenter = presenter # rubocop:disable Rails/HelperInstanceVariable
     end
```

Comment:
This is a false positive, as this isn't actually an instance variable in the helper itself.